docs/src/inputs/select/examples/exampleRequiredSelect.jsx

```diff
@@ -0,0 +1,29 @@
+import { isEmpty } from 'lodash';
+import React, { useState } from 'react';
+import {
+    Select,
+} from 'react-cm-ui';
+import { options } from '../constants';
+
+function ExampleRequiredSelect() {
+    const [selectedValue, setOption] = useState({});
+    function onSelect(selectedOption) {
+        setOption(selectedOption);
+    }
+
+    return (
+        <div>
+            <Select
+                id="block--required_select_id"
+                label="Select User"
```

Comment:
Should probably be "Select Option" since it is not using the "Users" custom options like some other examples.

Reply:
fixed.

Reply:
Resolving.